enhancements/kueue/openshift-kueue.md

```diff
@@ -0,0 +1,760 @@
+---
+title: openshift-kueue
+authors:
+  - kannon92
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - haircommander 
+  - rphillips
+  - varshaprasad96 #rhoai expert
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - mrunalp
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - joelspeed
+creation-date: 2025-02-19
+last-updated: 2025-02-19
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OCPSTRAT-1641
+see-also:
+  - "https://github.com/openshift/enhancements/pull/1736"
+replaces:
+  - "NA"
+superseded-by:
+  - "NA"
+---
+
+# Bringing Kueue into Openshift
+
+## Summary
+
+We would like to bring Kueue into the core platform as a managed operator.
+Kueue will be hosted as a Red Hat ecosystem operator in OperatorHub.
+
+## Motivation
+
+Kueue has wide applications across various projects in the openshift ecosystem. 
+We want to bring them as a core platform so we can better support them in various areas. 
+Openshift AI (RHOAI) uses kueue for AI workloads but we are also seeing autoscaling, multicluster and general batch uses. 
```

Comment:
nit: `but we are also seeing requets for....`

Reply:
updated.